### ISSUE TITLE
crystal tool xyz; like atom?

### ISSUE THREAD

**Issue by monouser7dig** [crystal-lang-tools/sublime-crystal#28]
[crystal lang examples in atom](https://crystal-lang.org/2015/09/05/tools.html)

sublime is always a bit behind atom in terms of user friendly tooling but do you have plans to enable these tooling options for sublime? (if this does not require too much hacking)

**Comment by faustinoaq** [comment]
@monouser7dig Yeah, we are working on it, see [Scry](https://github.com/crystal-lang-tools/scry)

Fixing some issues: https://github.com/crystal-lang-tools/scry/issues/34

and implementing new features: https://github.com/crystal-lang-tools/scry/pull/39

![bug03](https://user-images.githubusercontent.com/3067335/31861259-ab6e515a-b6ef-11e7-8826-e6950395bfe0.gif)
